src/components/home/Main.tsx

```diff
@@ -2,17 +2,20 @@ import React from 'react'
 import { ProfileCard } from '@/components/home/ProfileCard'
 import classes from '@/components/home/style.css'
 import useGetUsers from '@/components/home/useGetUsers'
+import { useRecoilValue } from 'recoil'
+import { userInfoState } from '@/atoms/userInfoAtom'
 
 export const Main = () => {
   const { Users, transferToProfile } = useGetUsers()
-  const myId = '3f328652-f4bb-4254-972a-d70489794a25'
+  const userInfo = useRecoilValue(userInfoState)
+  const myId = Object.keys(userInfo)[0]
 
   return (
     <div className={classes.container}>
       {Object.keys(Users)
         .filter((key) => key !== myId)
         .map((key) => {
-          return <ProfileCard key={key} User={Users[key]} transferToProfile={transferToProfile} />
+          return <ProfileCard key={key} Id={key} User={Users[key]} transferToProfile={transferToProfile} />
```

Comment:
細かいですが、camel caseだとすると、初めの文字は小文字なので、
```suggestion
          return <ProfileCard key={key} id={key} user={Users[key]} transferToProfile={transferToProfile} />
```
かなと思いました！(他にも同様の箇所があった気がします)

Reply:
@kan-mayoshi 
修正しました